### ISSUE TITLE
Artifact list

### ISSUE THREAD

**Issue by donutcake1** [marissandria/effbot#14]
Instead of leaving artifacts data in code, it would be better to use a .txt file of the following structure:

```<artifact_id>, <artifact_name>, <artifact_type>``` and so on...

Why it's better? You don't need to store it in code, plus, code will look shorter...

**Comment by marissandria** [comment]
Yeah I was planning on just building in a artifact CSV parser. Thanks for reminding me. :>
